app/(pages)/programs/protege-program/page.tsx

```diff
@@ -0,0 +1,61 @@
+import {Box, Stack, Typography} from '@mui/joy';
+import PageHeaderCard from "@/components/PageHeaderCard";
+import Image from 'next/image';
+import HeadingBodyText from '@/components/HeadingBodyText';
+
+export default function ProtegeProgram() {
+
+  const bodyText =
+  'The Protege Program is a mentoring program where female-identifying first-year engineering students will be grouped with a mentor studying engineering in their second year or higher to learn more about UNSW and engineering at UNSW. It is a great opportunity to make friends with other engineering girls and gain some useful advice from older students.'
+
+  return (
+
+    <Box>
+      <PageHeaderCard imagePath={'/programs/header.jpg'} pageTitle={'Programs'}/>
+      <HeadingBodyText heading='Protege Program' body={bodyText} color='graphite'/>
+
+      <Box sx={{ px: 10, mt: 5, mb: 20 }}>
+        <Stack direction={{ lg: 'column', xl: 'row-reverse' }} spacing={{ lg: 0, xl: 3 }} sx={{ margin: '0 auto' }} alignItems='center'>
+          <Stack direction='column' spacing={2} alignItems='center' sx={{ width: '50%' }}>
+            <Box width='583px'>
+              <Typography level='h4' sx={{ mb: 1 }}>
+                Meet other female engineers in your field
```

Comment:
instead of repeating code here for each section, you could create component i.e program section

Reply:
it may also be easier to implement the mobile responsiveness this way as well!

Reply:
if you want to take it a step further, you can also try using the map function to iterate over each bit of the program components section, it'll make your code a bit more maintainable and readable